discovery-infra/test_infra/utils.py

```diff
@@ -239,10 +239,12 @@ def wait_till_all_hosts_are_in_status(
         log.info("All nodes: %s", hosts)
         raise
 
+
 def wait_till_at_least_one_host_is_in_status(
```

Comment:
need to change the name, not it is not at least one, it is number of nodes right?

Reply:
not sure, default is 1, so it is represent its purpose

Reply:
ok